### PR TITLE
125 updated dependencies

### LINKED ISSUE #125: Update dependencies
### Description

Title says it all

### PR DESCRIPTION
**Pull request by junajan**
#### Summary
Resolves #125 

GreenKeepers PRs affected:
closes #105 closes #103 closes #111 closes #136 closes #92 closes #172 closes #124 closes #96 closes #52 closes #118 closes #151 closes #159 closes #158 closes #102 closes #108 closes #106 closes #123 closes #91 closes #110 closes #104 closes #119 closes #94 closes #97 closes #107 closes #101 closes #88 closes #93

### CODE REVIEW

File: types/sdk.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 /* Client */
 export type ClientRequest = {
   uri: string;
-  method: MethodType;
+  method: string;
```
Comment (emmenko) [inline types/sdk.js]: Why this change?

Comment (junajan) [inline types/sdk.js]: Because a `MethodType` is not defined.

Comment (emmenko) [inline types/sdk.js]: I get that, but can you investigate why? Because until now it was working.

Hint: it was implicitly defined within "flow" context. I guess in the latest versions this changed?

Comment (junajan) [inline types/sdk.js]: I will investigate it.

Comment (junajan) [inline types/sdk.js]: Originaly there was a `MethodType` loaded from here:
**v0.46.0 -** https://github.com/facebook/flow/blob/v0.46.0/lib/bom.js#L841

But in the latest version, it got deleted:
**v0.47.0 -** https://github.com/facebook/flow/blob/v0.47.0/lib/bom.js

Also, in the latest release, there is no notice about removed types: https://github.com/facebook/flow/releases/tag/v0.47.0

Comment (junajan) [inline types/sdk.js]: @emmenko - I added a missing `MethodType` here 9b3e187

**Comment by junajan** [comment]
@emmenko I added a yarn lockfile here 15b71fe0b8e2664b2e4226f0c47e0c9131d67de6

File: types/sdk.js (hunk 1 above)
Comment (emmenko) [inline types/sdk.js]: Awesome, thanks 👍

**Comment by junajan** [comment]
@emmenko - yep, working on it

**Comment by junajan** [comment]
@emmenko - with @wizzy25 we were not able to make our code working with Babel and Node v4 so we fixed wrong parts in our code here bbd1b60
